t/plugin/proxy-rewrite3.t

```diff
@@ -151,3 +151,52 @@ plugin_proxy_rewrite get method: GET
 --- response_body
 property "method" validation failed: matches none of the enum values
 done
+
+
+
+=== TEST 6: set route(rewrite method with headers)
```

Comment:
I think the test case should be "without headers"?

Reply:
There is a header in Line 173

Reply:
> I think the test case should be "without headers"?

at line173 config rewrite header